Include/Acidanthera/Library/OcVirtualFsLib.h

```diff
@@ -45,6 +47,32 @@ CreateVirtualFile (
   IN OUT EFI_FILE_PROTOCOL  **File
   );
 
+/**
+  Creates read-only EFI_FILE_PROTOCOL instance over a buffer allocated
+  from pool, but copies FileName internally. On success FileData ownership is transferred
```

Comment:
1. Is `ModificationTime` also copied? I guess it is, so better to remove `, but copies FileName internally` and add `All the other fields are copied.` right after `upon closing EFI_FILE_PROTOCOL`.

2. To make copying even more explicit, please add `CONST` parameter to the not owned pointer arguments (`FileName`, `ModificationTime`).

3. File should be just out, as we do not read its value, but only write. This is in fact true for several other functions in this file.